util/throttle.cc

```diff
@@ -140,6 +142,13 @@ ThrottleThread(
             tot_compact+=gThrottleData[loop].m_Compactions;
         }   // for
 
+        // capture current state of level-0 and other levels' backlog
+        gThrottleData[replace_idx].m_Backlog=gCompactionThreads->m_WorkQueueAtomic;
+        gPerfCounters->Add(ePerfThrottleBacklog1, gThrottleData[replace_idx].m_Backlog);
+
+        gThrottleData[0].m_Backlog=gLevel0Threads->m_WorkQueueAtomic;
+        gPerfCounters->Add(ePerfThrottleBacklog0, gThrottleData[0].m_Backlog);
+
 	// non-level0 data available?
```

Comment:
fixed

> On Aug 9, 2015, at 10:06 PM, Paul A. Place <notifications@github.com> wrote:
> 
> In util/throttle.cc <https://github.com/basho/leveldb/pull/162#discussion_r36597820>:
> 
> > @@ -140,6 +142,13 @@ ThrottleThread(
> >              tot_compact+=gThrottleData[loop].m_Compactions;
> >          }   // for
> >  
> > +        // capture current state of level-0 and other levels' backlog
> > +        gThrottleData[replace_idx].m_Backlog=gCompactionThreads->m_WorkQueueAtomic;
> > +        gPerfCounters->Add(ePerfThrottleBacklog1, gThrottleData[replace_idx].m_Backlog);
> > +
> > +        gThrottleData[0].m_Backlog=gLevel0Threads->m_WorkQueueAtomic;
> > +        gPerfCounters->Add(ePerfThrottleBacklog0, gThrottleData[0].m_Backlog);
> > +
> >  	// non-level0 data available?
> Have tabs instead of spaces in the following lines: 152, 176-180, 184, and 191
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/basho/leveldb/pull/162/files#r36597820>.
>